lib/signs/utilities/messages.ex

```diff
@@ -98,13 +98,21 @@ defmodule Signs.Utilities.Messages do
     if has_single_source_list?(sign) do
       {nil, Content.Message.Empty.new()}
     else
-      source_config =
+      {source_config, headway_group} =
         if line == :top do
           %Signs.Realtime{source_config: {top_line_source, _}} = sign
-          top_line_source
+          headway_group = Signs.Utilities.SourceConfig.default_headway_group(sign.headway_group)
+          {top_line_source, headway_group}
         else
           %Signs.Realtime{source_config: {_, bottom_line_source}} = sign
-          bottom_line_source
+
+          headway_group =
+            case sign.headway_group do
+              [_, bottom_headway_group] -> bottom_headway_group
```

Comment:
This is giving a "pattern can never match" warning for some reason, but I'm pretty sure it's just dialyzer being wrong 🤷

Reply:
I'm getting this too in my editor, but not when running dialyzer manually. Is this a known issue?

Reply:
I don't know if it's a documented issue off the top of my head, but I know that ElixirLS can be flaky when it comes to dialyzer.